server/src/test/java/org/apache/druid/server/coordinator/duty/CompactSegmentsTest.java

```diff
@@ -1918,10 +1956,13 @@ private void addMoreData(String dataSource, int day)
 
   private List<DataSourceCompactionConfig> createCompactionConfigs()
   {
-    return createCompactionConfigs(null);
+    return createCompactionConfigs(null, null);
   }
 
-  private List<DataSourceCompactionConfig> createCompactionConfigs(@Nullable Integer maxNumConcurrentSubTasks)
+  private List<DataSourceCompactionConfig> createCompactionConfigs(
```

Comment:
Instead of doing this, have two separate methods: `createCompactionConfigsForNative`, `createCompactionConfigsForMsq`

Reply:
Have retained the added function but added the suggested layers on top of it, so that the entire body need not be repeated.